src/TLuaInterpreter.cpp

```diff
@@ -346,8 +346,8 @@ int TLuaInterpreter::raiseEvent( lua_State * L )
 {
     Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
     if( ! pHost ) {
-        lua_pushstring( L, tr( "raiseEvent: NULL Host pointer - something is wrong!" )
-                        .toUtf8().constData() );
+        lua_pushstring(L, QStringLiteral("raiseEvent: NULL Host pointer - something is wrong!")
+                       .toLatin1().constData());
```

Comment:
Why are we using Qt types here at all? `lua_pushstring(L, "raiseEvent: NULL Host pointer - something is wrong!");` will work exactly the same way in this case, so using Qt types just makes things more complex for no benefit.